parser.js

```diff
@@ -0,0 +1,113 @@
+import { RootNode, Node, TextNode } from './class.js';
+
+const removeLineBreaks = (str) => str.replace(/(\r\n|\n|\r)/gm, '');
+
+/* @params attributesToBeParsed : ['class="container"'] */
+/* @return attributes : [['class', 'container']] */
+const parseAttributes = (attributesToBeParsed) => {
+  return attributesToBeParsed.map((v) => {
+    let [key, value] = v.split('=');
```

Comment:
```suggestion
  return attributesToBeParsed.map((attribute) => {
    let [key, value] = attribute.split('=');
```

바로 밑에 `value`가 나와서 `v`가 뜻하는게 뭔지 안와닿는거 같아서, `attribute`와 같이 명시적으로 써보면 어떨까 제안해봅니다 :+1:

Reply:
제안해주신대로 명시적으로 작성하는 것 훨씬 좋은 것 같아 반영했습니당 :) 👍👍👍

Reply:
📌 관련커밋: f38aa59